scripts/postinstall.ts

```diff
@@ -98,7 +98,7 @@ function buildDownloadContextFor(rawPlatform: DependencyPlatform, depVersions: D
   const resourcesDir = path.join(process.cwd(), 'resources');
   const downloadContext: DownloadContext = {
     versions:          depVersions,
-    dependencyPlaform: rawPlatform,
+    dependencyPlatform: rawPlatform,
```

Comment:
a linter is going to scream at me

Reply:
There are three instances of this typo in the code base.

Reply:
Hmm, I'll have to figure out what happened here.

Reply:
Yeah, they're all covered:
https://github.com/rancher-sandbox/rancher-desktop/pull/5030/commits/e3889f6dc5371cb362ecbd6b196f0be39f2a24e0

This was the only time where the whitespace appeared to matter.